scripts/ASTImportTest.sh

```diff
@@ -14,90 +45,91 @@ SOLIDITY_BUILD_DIR=${SOLIDITY_BUILD_DIR:-${REPO_ROOT}/build}
 SOLC=${SOLIDITY_BUILD_DIR}/solc/solc
 SPLITSOURCES=${REPO_ROOT}/scripts/splitSources.py
 
+# shellcheck source=scripts/common.sh
+source "${REPO_ROOT}/scripts/common.sh"
+
 SYNTAXTESTS_DIR="${REPO_ROOT}/test/libsolidity/syntaxTests"
 ASTJSONTESTS_DIR="${REPO_ROOT}/test/libsolidity/ASTJSON"
-NSOURCES="$(find "$SYNTAXTESTS_DIR" -type f | wc -l)"
-
-# DEV_DIR="${REPO_ROOT}/../tmp/contracts/"
-# NSOURCES="$(find $DEV_DIR -type f | wc -l)" #TODO use find command
 
 FAILED=0
 UNCOMPILABLE=0
 TESTED=0
 
 if [[ "$(find . -maxdepth 0 -type d -empty)" == "" ]]; then
-    echo "Test directory not empty. Skipping!"
-    exit 1
+    fail "Test directory not empty. Skipping!"
 fi
 
-# function tests whether exporting and importing again leaves the JSON ast unchanged
+function ast_import_export_equivalence
+{
+    local sol_file="$1"
+    local input_files="$2"
+    # save exported json as expected result (silently)
+    $SOLC --combined-json ast --pretty-json --json-indent 4 "${input_files}" > expected.json 2> ${REDIRECT_TO}
```

Comment:
Ok.. because of the description of #8185 I was just "replacing" `/dev/null` here - but looking at this now it makes not that much sense..